guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p><code>however if the remaining time left to process the network is less than than the expected time</code> to </p><p><code>however if the remaining time left to process the network is less than the expected time</code> </p><p>Can you clarify what expected time means? </p><p> </p><p>We can reword this statement <code>In the second half it checks for two best performing networks.</code></p><p> </p><p><code>than than the expected time</code> . Maybe we can reword   <code>expected </code> time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>